packages/cli/core/src/commands/atomic/init.ts

```diff
@@ -0,0 +1,82 @@
+import {CLICommand} from '@coveo/cli-commons/command/cliCommand';
+import {Config} from '@coveo/cli-commons/config/config';
+import {Before} from '@coveo/cli-commons/decorators/before';
+import {Beta} from '@coveo/cli-commons/decorators/beta';
+import {UnknownError} from '@coveo/cli-commons/errors/unknownError';
+import {Flags} from '@oclif/core';
+import inquirer from 'inquirer';
+import {
+  atomicAppPreconditions,
+  atomicLibPreconditions,
+  createAtomicApp,
+  createAtomicLib,
+} from '../../lib/atomic/createAtomicProject';
+
+export default class AtomicInit extends CLICommand {
+  public static description =
+    'Scaffold a project to work with Coveo Atomic Framework';
+
+  public static examples = [
+    '<%= config.bin %> <%= command.id %> --type=app myAwesomeSearchPage',
+    '<%= config.bin %> <%= command.id %> --type=lib myCustomAtomicComponentsLibrary',
+  ];
+
+  public static flags = {
+    type: Flags.string({
+      description: 'the kind of project to initialize',
+      options: ['app', 'application', 'lib', 'library'],
```

Comment:
Why offer "aliases" for the option name ?

Reply:
I think it's a nice way to have an abbreviated flag option, while still documenting what they mean fully.
(granted ideally I'd prefer to be able to have `app[lication]` instead of `app, application` in the doc)